spec/selection.spec.js

```diff
@@ -41,6 +41,19 @@ describe('Selection', function () {
     })
   })
 
+  describe('removeChars()', function () {
+
+    it('removes multiple characters', function () {
```

Comment:
it would be nice to have a test that the characters are not removed in the tag `<a href=""></a>`

Reply:
I didn't add one as the method works on the DOM so these characters don't really exist from that perspective. So I think it's fine for now. We don't currently have methods that work on serialised HTML in the `cursor` or `selection` classes.